server/lib/configuration.js

```diff
@@ -217,6 +225,12 @@ const conf = module.exports = convict({
     env: 'FLOW_ID_KEY',
     format: String
   },
+  flow_metrics_disabled: {
+    default: false,
+    doc: 'disable flow metrics output to stderr',
+    env: 'DISABLE_FLOW_METRICS_STDERR',
```

Comment:
Since the previous two environment variables start with `FLOW_`, let's keep the same prefix. Can you remove the `_STDERR` suffix too? Maybe just `FLOW_METRICS_DISABLED`.